src/apps/common/win/getopt.h

```diff
@@ -77,12 +77,14 @@ struct option /* specification for a long form option...	*/
   int val;          /* its associated status value		*/
 };
 
-enum /* permitted values for its `has_arg' field...	*/
-{
-  no_argument = 0,   /* option never takes an argument	*/
+// clang-format off
+//	different clang-format versions have different opinions on this and prevent PRs to succeed
+enum                 /* permitted values for its `has_arg' field...	*/
+{ no_argument = 0,   /* option never takes an argument	*/
```

Comment:
Ehmm, the PR failed because ’make lint’ was not happy with it, when it got linted on ubuntu 20.04. If fixed for the linter there, the ’make lint’ for ubuntu 22.04 failed. So a deadlock for PRs (wondering, how all the other PRs got through ;-)).